content/en/universal_service_monitoring/setup.md

```diff
@@ -730,48 +727,36 @@ If you use load balancers with your services, enable additional cloud integratio
    service_monitoring_config:
      enabled: true
    ```
-[1]: /agent/basic_agent_usage/windows/?tab=commandline
-{{% /tab %}}
-
-{{< /tabs >}}
-
-## Additional configuration
-
-The following systems or services require additional configuration:
-
-{{< collapse-content title="Non-containerized services on Linux" level="h4" >}}
-<div class="alert alert-info">
-Universal Service Monitoring is available in <strong>beta</strong> to monitor services running bare-metal on Linux virtual machines.
-</div>
+**For non-IIS services:**
 
 Requires Agent version 7.42 or greater.
```

Comment:
we might want to highlight that versions prior to 7.57 but post 7.42 need the configuration enabled

Reply:
thanks edited to mention this